seqr/views/apis/anvil_workspace_api.py

```diff
@@ -0,0 +1,118 @@
+"""APIs for management of projects related to AnVIL workspaces."""
+
+import logging
+import json
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import redirect
+
+from seqr.models import Project
+from seqr.views.utils.json_to_orm_utils import create_model_from_json
+from seqr.views.utils.json_utils import create_json_response
+from seqr.views.utils.file_utils import load_uploaded_file
+from seqr.views.utils.terra_api_utils import is_google_authenticated, user_get_workspace_access_level, add_service_account
+from seqr.views.apis.individual_api import _add_or_update_individuals_and_families
```

Comment:
I know we do it in to many places in the code, but its really bad to import protected methods in python (methods that start with `_`). You should actually move this function to the `individual_utils` file and rename it without the underscore

Reply:
I will update it a little later.